tests/network/l2_bridge/conftest.py

```diff
@@ -42,6 +42,7 @@
 VMB_MPLS_LOOPBACK_IP = f"{random_ipv4_address(net_seed=6, host_address=1)}/32"
 VMB_MPLS_ROUTE_TAG = 200
 DHCP_INTERFACE_NAME = "eth3"
+UNIQUE_CLIENT_ID = "dhcp-client-id"
```

Comment:
Is it unique?
Consider randomizing it, like we do with VM names. It should help with cases where two clusters are running on the same VLAN with the DHCP server active.